playwright-tests/tests/payments/create-payment-request.spec.js

```diff
@@ -303,8 +303,10 @@ test.describe("admin connected", function () {
 
     await clickCreatePaymentRequestButton(page);
 
-    expect(await page.getByTestId("proposal-title").inputValue()).toBe("");
-    expect(await page.getByTestId("proposal-summary").inputValue()).toBe("");
+    if (daoAccount === "infinex.sputnik-dao.near") {
```

Comment:
Are these changes in the test related to the referenced issues in the PR? I don't see that they are connected to voting policy. Should be described in the PR, so that it is possible to understand what has been changed here.

Reply:
so I made [this](https://github.com/NEAR-DevHub/neardevhub-treasury-dashboard/pull/178/files#diff-fee2f2d99d350199c36da8967ec42443e2093086ff3912b0e449210646c7c28dR60
) change, which resets the create payment form to default on closure, that's why needed to fix this test.